lib/linguist/heuristics.rb

```diff
@@ -380,6 +380,24 @@ def call(data)
       end
     end
 
+    disambiguate ".pod" do |data|
+      if /^[\s&&[^\n]]*=\w+$/.match(data)
+        if /^[\s&&[^\n]]+=(begin|cut|pod)/.match(data)
+          Language["Pod 6"]
+        elsif /^=(pod|cut)/.match(data)
```

Comment:
Maybe allow it to include spaces before =pod or =cut in case the document is formatted improperly? Optional, since that's not actual valid Pod, but could occur on accident.

Reply:
IMO if someone indents a Perl 5 pod directive in a .pod file that's their own fault, it doesn't need to complicate this heuristic.